Makefile

```diff
@@ -23,13 +23,12 @@ STATICCHECK_IGNORE = \
 
 DOCKER_IMAGE_NAME       ?= prometheus
 
-ifdef DEBUG
-	bindata_flags = -debug
-endif
+VFSGENDEV := $(FIRST_GOPATH)/bin/vfsgendev
 
 .PHONY: assets
-assets:
+assets: $(VFSGENDEV)
 	@echo ">> writing assets"
-	@$(GO) get -u github.com/jteeuwen/go-bindata/...
-	@go-bindata $(bindata_flags) -pkg ui -o web/ui/bindata.go -ignore '(.*\.map|bootstrap\.js|bootstrap-theme\.css|bootstrap\.css)'  web/ui/templates/... web/ui/static/...
-	@$(GO) fmt ./web/ui
+	cd $(PREFIX)/web/ui && go generate
+
+$(VFSGENDEV):
+	cd $(PREFIX)/vendor/github.com/shurcooL/vfsgen/ && go install ./cmd/vfsgendev/...
```

Comment:
I'm not sure we should be modifying the user's environment like this

Reply:
That's a good point. I've replaced `vfsgen`by a custom helper program that does the same (just a couple of lines).